eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -205,7 +205,7 @@ function Upload-Blobs
     if ($ReleaseTag) {
         foreach ($htmlFile in (Get-ChildItem $DocDir -include *.html -r)) 
         {
-            $fileContent = Get-Content -Path $htmlFile
+            $fileContent = Get-Content -Path $htmlFile -Raw
             $updatedFileContent = $fileContent -replace $RepoReplaceRegex, "`${1}$ReleaseTag"
             if ($updatedFileContent -ne $fileContent) {
                 Set-Content -Path $htmlFile -Value $updatedFileContent
```

Comment:
@sima-zhu since you are now using -Raw have you verified that the Set-Content works correctly? I think you will need to now pass the "-NoNewLine" option to it as well otherwise it will add a new line at the end of the file.

Reply:
Did not verify newline, but double check with the all release tag works just fine. Will do more check here and submit a new PR if it is a miss.

Reply:
I just tested it locally, it actually add new line in the scripts. Thanks for spotting the issue. Will submit another PR.